app-office/texstudio/additional-files/texliveinstall.sh

```diff
@@ -0,0 +1,36 @@
+#/bin/sh
```

Comment:
@jmairboeck added a small script here to install the required base packages, not sure if it's OK or maybe include something (if there isn't one already) in the main texlive package?